File: box-server/box-domain/src/main/java/com/boxai/domain/config/SystemConfig.java
package com.boxai.domain.config;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
public class SystemConfig {

    private String configKey;
    private String configValue;
    private String description;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: box-admin-web/src/api/systemConfig.ts
import http, { type Result } from './http'

export interface SystemConfigVO {
  configKey: string
  configValue: string
  description?: string
}

export function fetchSystemConfigs() {
  return http.get<Result<SystemConfigVO[]>>('/system/config')
}

export function upsertSystemConfig(payload: {
  configKey: string
  configValue?: string
  description?: string
}) {
  return http.put<Result<SystemConfigVO>>('/system/config', payload)
}
